src/main/java/org/sonarsource/scanner/cli/SystemInfo.java

```diff
@@ -19,7 +19,19 @@
  */
 package org.sonarsource.scanner.cli;
 
+import java.util.HashSet;
+import java.util.Set;
+import java.util.regex.Pattern;
+import java.util.stream.Collectors;
+import java.util.stream.Stream;
+
 class SystemInfo {
+  private static final Set<String> SENSITIVE_JVM_ARGUMENTS = Stream.of(
```

Comment:
No need to create a stream, we could use `Set.of`.
The added benefit is that the set will be ummutable (not the case here!)

Reply:
I didn't realize it was mutable, to be fair I took the shortcut thinking of `final` => immutable. But this is just about affectation, not the object itself, now I get that ;) 

Also I didn't use `Set.of()` at first because the project was still compiled with Java 8. As the doc is clear that scanners minimal version is Java 11, I went ahead to use Java 11, so I could use `Set.of()`.
Let me know if you think this Java config change could be an issue, I don't think so.